templates/admin/release_report_detail.html

```diff
@@ -346,5 +356,30 @@ <h4>
       const chart = new ApexCharts(document.querySelector("#top-committed-libraries-chart"), options);
       chart.render();
     </script>
+    <script>
+      // Use fitText to resize text to fit its container.
+      // Starts at MAX_FONT_SIZE and tries smaller sizes until it fits or hits MIN_FONT_SIZE.
```

Comment:
while this works, using a css container would be a better use case in the future:
https://www.youtube.com/watch?v=DHj7JhH8ins

Reply:
I was not sure if there was a pure CSS method to do this. Is container queries the answer?

> Container queries enable you to apply styles to an element based on the size of the element's container

If this is all they do, it doesn't seem like enough.  The container is always the same size, it is the text content that differs.  I'd love to see a CSS only version if it works, though.  My cursory search didn't bring me anything fruitful.